app/policies/article_policy.rb

```diff
@@ -47,15 +47,14 @@ def admin_unpublish?
   #       appears with the pull request)
   #
   # @note For backwards compatability purposes, we're not checking if there's a user.
-  def new?
-    true
-  end
 
   def create?
     require_user!
-    !user_suspended?
+    user.any_admin? && !user_suspended?
```

Comment:
Let's not include this for the moment, I'm working on the policy behavior with a feature flag and should have that up in short order.

With this line we break lots of existing specs, which might be a distraction from the implementation details.